phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+    level: max
+    paths:
+        - src
+    excludePaths:
+        - src/M6Web/Bundle/LogBridgeBundle/Config/*
+        - src/M6Web/Bundle/LogBridgeBundle/DependencyInjection/*
+        - src/M6Web/Bundle/LogBridgeBundle/EventDispatcher/BuilderEvent.php
+        - src/M6Web/Bundle/LogBridgeBundle/Tests/*
```

Comment:
If there are not a lot of different errors, could you ignore the message with a regex pattern instead of excluding an entire directory?
IMO it will be easier to debug and fix these errors later.

Reply:
Done : the list is quite huge... but I suppose it's fine ?

Reply:
Reduced the list to the `DependencyInjection` folder